urllib3/util/timeout.py

```diff
@@ -11,11 +11,10 @@
 _Default = object()
 
 
-def current_time():
-    """
-    Retrieve the current time. This function is mocked out in unit testing.
-    """
-    return time.time()
+# Use time.monotonic if available.
+current_time = time.time
+if hasattr(time, "monotonic"):
```

Comment:
Why not do something like:

```py
current_time = getattr(time, 'monotonic', time.time)
```

This avoids using hasattr, and keeps the logic nice and simple